server/src/main/java/org/apache/cassandra/sidecar/acl/authorization/FeaturePermission.java

```diff
@@ -0,0 +1,91 @@
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.acl.authorization;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.apache.cassandra.sidecar.acl.authorization.BasicPermissions.CREATE_RESTORE_JOB;
+import static org.apache.cassandra.sidecar.acl.authorization.BasicPermissions.CREATE_SNAPSHOT;
+import static org.apache.cassandra.sidecar.acl.authorization.BasicPermissions.DELETE_RESTORE_JOB;
+import static org.apache.cassandra.sidecar.acl.authorization.BasicPermissions.DELETE_SNAPSHOT;
+import static org.apache.cassandra.sidecar.acl.authorization.BasicPermissions.DELETE_STAGED_SSTABLE;
+import static org.apache.cassandra.sidecar.acl.authorization.BasicPermissions.EDIT_RESTORE_JOB;
+import static org.apache.cassandra.sidecar.acl.authorization.BasicPermissions.IMPORT_STAGED_SSTABLE;
+import static org.apache.cassandra.sidecar.acl.authorization.BasicPermissions.READ_RESTORE_JOB;
+import static org.apache.cassandra.sidecar.acl.authorization.BasicPermissions.READ_RING_KEYSPACE_SCOPED;
+import static org.apache.cassandra.sidecar.acl.authorization.BasicPermissions.READ_SCHEMA;
+import static org.apache.cassandra.sidecar.acl.authorization.BasicPermissions.READ_SCHEMA_KEYSPACE_SCOPED;
+import static org.apache.cassandra.sidecar.acl.authorization.BasicPermissions.READ_SNAPSHOT;
+import static org.apache.cassandra.sidecar.acl.authorization.BasicPermissions.READ_TOPOLOGY;
+import static org.apache.cassandra.sidecar.acl.authorization.BasicPermissions.STREAM_SNAPSHOT;
+import static org.apache.cassandra.sidecar.acl.authorization.BasicPermissions.UPLOAD_STAGED_SSTABLE;
+import static org.apache.cassandra.sidecar.acl.authorization.CassandraPermissions.MODIFY;
+import static org.apache.cassandra.sidecar.acl.authorization.CassandraPermissions.SELECT;
+import static org.apache.cassandra.sidecar.acl.authorization.ResourceScopes.TABLE_SCOPE;
+
+/**
+ * Enumerates a list of feature level permissions that Sidecar recognizes and honors.
+ */
+public enum FeaturePermission
+{
+    ANALYTICS_READ_DIRECT("ANALYTICS:READ_DIRECT",
+                          READ_RING_KEYSPACE_SCOPED,
+                          READ_SCHEMA_KEYSPACE_SCOPED,
+                          CREATE_SNAPSHOT,
+                          READ_SNAPSHOT,
+                          DELETE_SNAPSHOT,
+                          STREAM_SNAPSHOT,
+                          new StandardPermission(SELECT.name(), TABLE_SCOPE)),
+
+    ANALYTICS_WRITE_DIRECT("ANALYTICS:WRITE_DIRECT",
+                           READ_SCHEMA_KEYSPACE_SCOPED,
+                           READ_TOPOLOGY,
+                           UPLOAD_STAGED_SSTABLE,
+                           IMPORT_STAGED_SSTABLE,
+                           DELETE_STAGED_SSTABLE,
+                           new StandardPermission(MODIFY.name(), TABLE_SCOPE)),
+
+    ANALYTICS_WRITE_S3_COMPAT("ANALYTICS:WRITE_S3_COMPAT",
+                              READ_SCHEMA,
```

Comment:
Does this need to be `READ_SCHEMA_KEYSPACE_SCOPED` ?

Reply:
Good catch, updated it